app/javascript/react/components/CuisineTile.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+
+class CuisineTile extends Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
```

Comment:
Please remove empty state and, in fact, if you don't need state these should use `const` syntax instead!